.readthedocs.yaml

```
@@ -5,6 +5,10 @@
 # Required
 version: 2
 
+formats:
+  - epub
+  - pdf
+
```

What about building all formats?
```suggestion
formats: all
```